package.json

```
@@ -61,11 +61,12 @@
 	},
 	"scripts": {
 		"a11y": "pa11y-ci --reporter=pa11y-ci-reporter-html --sitemap $npm_config_url/sitemap.xml",
-		"build": "cross-env NODE_ENV=production wp-scripts build --config webpack.prod.js",
+    	"build:blocks": "for dir in blocks/*/; do (cd \"$dir\" && npm run build); done",
```

```suggestion
    	"build:blocks": "for dir in blocks/*/; do if [ -d \"$dir\" ]; then (cd \"$dir\" && npm run build); fi done",
```
I think adding this `if` check fixes running the build with an empty `blocks` folder.